src/clj/rems/poller/common.clj

```diff
@@ -25,7 +25,7 @@
         (throw (InterruptedException.)))
       (try
         ;; TODO: add proper monitoring for pollers and caches
-        (log/info name-kw "processing event" (:event/id event))
+        (log/info name-kw "processing event" (:event/id event) (:event/type event))
```

Comment:
consider `(select-keys event [:event/id :event/type])`

Reply:
There is no guaranteed order in logs then?

Reply:
ok good point